documentation/jetty-documentation/src/main/asciidoc/operations-guide/deploy/deploy-hot-static.adoc

```diff
@@ -27,9 +27,9 @@ The following command line disables _hot_ deployment by specifying the `jetty.de
 $ java -jar $JETTY_HOME/start.jar jetty.deploy.scanInterval=0
 ----
 
-To make _static_ deployment persistent, you need to edit the `deploy` module configuration file, `$JETTY_BASE/start.d/deploy.ini`, uncomment the module property `jetty.deploy.scanInterval` and change its value to `0`:
+To make _static_ deployment persistent, you need to edit the appropriate `<env>-deploy` module configuration file, `$JETTY_BASE/start.d/<env>-deploy.ini` (eg: `ee10-deploy.ini`), uncomment the module property `jetty.deploy.scanInterval` and change its value to `0`:
```

Comment:
The text above this is wrong, according to the conversation we had yesterday, where the conclusion was that jetty should _not_ hot deploy by default. So the default `jetty.deploy.scanInterval` needs to be 0, and the text changed to say that it will be static deployment by default, and if you want hot deployment you need to edit the appropriate `${ee}-deploy.ini` and change the value of `jetty.deploy.scanInterval` to a positive integer.

Reply:
The discussion ended at jetty-home being at 1 or 10, and embedded remaining at 0.
With documentation updated.